documentation/src/main/asciidoc/topics/con_xsite_concurrent_writes.adoc

```diff
@@ -0,0 +1,24 @@
+[id='conflicts-{context}']
+= Conflict Resolution with Cross-Site Replication
+Conflicting entries can occur with Active/Active site configurations if clients
+write to the same entries at the same time but at different sites.
+
+For example, client A writes to "k1" in **LON** at the same time that client B
+writes to "k1" in **NYC**. In this case, "k1" has a different value in **LON**
+than in **NYC**. After replication occurs, there is no guarantee which value
+for "k1" exists at which site.
+
+{brandname} uses the metadata of each key to detect conflicts during a backup.
```

Comment:
don't use the word `metadata` here since the user can put metadata with the value.
you can say "uses versioning to detect conflicting updates" or something similar.

Reply:
ah, ok. I didn't want to be too specific and say "vector clock" but I've updated. hope the changes are fine.